core/src/main/java/tech/tablesaw/io/FileReader.java

```diff
@@ -94,6 +94,12 @@ public String[] getColumnNames(
     if (options.header()) {
 
       String[] headerNames = parser.parseNext();
+      
+      if (headerNames == null) {
+        // no header because file is empty
+        // return empty String[]
+        return new String[0];
```

Comment:
the comment basically just duplicates the return line. if the return is a bit unclear it might be clearer written differently
```suggestion
        return new String[] {};
```